README.md

```diff
@@ -57,7 +57,7 @@ Linux amd64 Tags
 | `3.0`, `3.0-powershell6`                       | [Dockerfile](host/3.0/buster/amd64/powershell/powershell6.Dockerfile)            | Debian 10  |
 | `3.0-slim`, `3.0-powershell6-slim`             | [Dockerfile](host/3.0/buster/amd64/powershell/powershell6-slim.Dockerfile)       | Debian 10  |
 | `3.0-appservice`, `3.0-powershell6-appservice` | [Dockerfile](host/3.0/buster/amd64/powershell/powershell6-appservice.Dockerfile) | Debian 10  |
-
+| `3.0-powershell6-core-tools` | [Dockerfile](host/3.0/buster/amd64/powershell/powershell6-core-tools.Dockerfile) | Debian 10  |
```

Comment:
Not necessarily for this PR, but we will probably want to rethink the naming convention. The "6" in the PowerShell image names will not make much sense soon, as the Core Tools will include both PS6 and PS7 worker flavors, and the image building scripts just take the entire `/workers/powershell` content, which seems to be the right thing to do. But `/workers/powershell` will have `6` and `7` subfolders in Functions 3.0, and exactly the same will happen if we want to introduce a new image for "6".

I would just drop the version from the name: `3.0-powershell-core-tools`.

Alternatively, we could remove either `/workers/powershell/6` or `/workers/powershell/7` subfolders just for the sake of having two smaller images. But this would be a strange thing to do because the PowerShell version for Function apps is supposed to be determined by site config or environment variables.

@ahmelsayed: FYI. Please let us know if you have any ideas on this.

Reply:
Moving forward, the versioning story for PowerShell may become more interesting. For example, it is quite possible that we will add PowerShell 7.1 side-by-side with 7.0, etc., which will force us introduce more slightly different images. This sounds like one more reason to decouple image building scripts from PowerShell versioning, and leave the versioning story up to Core Tools, Functions Host, and later perhaps encapsulate it in the PowerShell worker entirely.

Reply:
I love the idea of the side-by-side scenario. That helps us to provide stable versions of the images rather than overwrite the major version with new revision images. It also help customer to use different version when something happened to the new image. Let's discuss with @ahmelsayed .

Reply:
one of the main motivations of having an image per version is to reduce the overall size of the image. with side-by-side support, do you mean 1 function app with both PowerShell 7.0 and PowerShell 7.1 functions that will load the different runtimes? 

The size might not be as sensitive for the core-tools images, though smaller images are much faster to pull on slower connections

Reply:
@ahmelsayed By side-by-side, I mean the same approach that we already use for PS6 (in fact 6.2.4) and PS7 (in fact 7.0.0), we'll just add one more item to this list. At any given time, a PS app will use exactly one runtime version, so we will never have to load them _simultaneously_. However, the core tools user expects that the app can be switched to another PS version by changing the FUNCTIONS_WORKER_RUNTIME_VERSION app setting or environment variable. If we go with separate images, the user will have to know that this is not enough, and they will need to select a different image as well. On the other hand, we can set FUNCTIONS_WORKER_RUNTIME_VERSION on the image itself, so the user will just have to select a proper image, which is good.

I understand the size point, though. Currently, the PS6 worker is ~97 MB and the PS7 worker is ~56 MB (unzipped). @TsuyoshiUshio, what is your current image size (I just want to understand the relative delta)?

So, we are having a tradeoff: increased image size vs. the overhead of maintaining a separate image per PowerShell version (we'll have to add a new image when we add a new PowerShell version, and we'll have to update the image building scripts if the directory structure within the PowerShell worker ever changes). Technically, we can go either way. I slightly prefer reducing the complexity, unless the image size increase is prohibitive.

Reply:
@AnatoliB 
For the current one, 3.24 GB.   For core-tools, it is mainly used for devcontainer and ci. It is slight different from other images, size doesn't matter much. However, user experience is important. In terms of the dev container, that difference might not be very big deal.  (In the near, future, people might ask us a slim image for CI though).